roles/setup_server/tasks/main.yaml

```diff
@@ -26,8 +26,9 @@
 
 - name: RHEL8 specific setup
```

Comment:
This is supposed to be specifically for RHEL8, I don't think Fedora should be included in this conditional block